### ISSUE TITLE
Possible docs additions to Wiki, Reference and ReadMe

### ISSUE THREAD

**Issue by tojon** [taskcluster/unified-logviewer#32]
In digging around to try to find out more about unified-logviewer, I was unable to find references to it in the TaskCluster [Components](https://wiki.mozilla.org/TaskCluster/Components) wiki or in the TaskCluster [Reference](https://docs.taskcluster.net/reference) core services documentation. Perhaps it might be useful to have an entry for them there, if one or both is applicable.

If so, it would be good to then add a couple of lines to reference that documentation location from the ReadMe.

**Comment by eliperelman** [comment]
I totally agree that we should document this properly. Right now this repo is in a state of limbo, as there are few things that will happen that will affect docs:

1. Upgrade to Neutrino v5
2. Remove from the TaskCluster organization and move to the "web-org".
3. Change this from being an iframe drop-in to an importable React component. We can create another iframe-able version of this if we desire, but we really need an importable version of the logviewer.

Once these are done, we can flesh out proper docs!

**Comment by eliperelman** [comment]
Published the first version of a standalone React component:

https://github.com/mozilla-rpweb/react-lazylog
